Portfolio: finish mobile version

DianaBeki opened the pull request:
- Used ul and li in "Work section", To include Html, Css and Javascript.
- Used flexbox to layout items in specify class.
- Update README.md.
- Linters config.
- Added background to all cards item in index.html file.
- Added hover effect to buttons in CSS file.

vickymarz commented on index.html:
- [ ] KIndly reduce the spacing between the content of your project cards to match your Figma design 👍🏽 

Figma Design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/198849937-9508301a-5acd-468c-bfc4-63d9ce7268c1.png)

Your Design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/198849954-dae904b1-b28a-4d81-b166-1a27ecc4c6db.png)

vickymarz commented on index.html:
- [ ] I will strongly recommend you wrap your `about me` title in a heading tag apart from `<h1>` as you should only have a single `<h1>` tag in your HTML file. You can use `<h2>` tag

e.g `<h2>About Myself</h2>`

vickymarz commented on index.html:
- [ ] Kindly increase the font weight of your about me title to match the figma design

FIgma Design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/198850094-5ae4aff5-81c4-4ecb-a203-a4069d5b5c67.png)

Your Design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/198850122-609929cc-3053-4a61-b35a-9b418f97ce32.png)

vickymarz commented on index.html:
- [ ] Kindly ensure that the color of your `LET’S CONNECT` matches the figma design

Figma Design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/198850200-96a99f1d-f427-4fdc-aef7-36be820894b4.png)

Your Design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/198850208-8145ddb9-c04c-419e-bb71-0d4e9263258f.png)

vickymarz commented on index.html:
- [ ] Kindly reduce the font-weight of your list of technologies to match the Figma design 👍🏽

Figma Design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/198850281-c501723e-a5ff-4f4a-93e7-2e84c42d28e7.png)

Your Design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/198850295-b664ac46-3857-4d77-bed3-588146e68810.png)

vickymarz reviewed (changes requested):
Hi @DianaBeki, 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Required Changes ♻️
_Check the comments under the review._
<img src="https://media2.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif"/>

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

DianaBeki commented:
Adjusted the following

- Reduced  spacing in the content cards
- Changed font weight for about myself section
- Changed the let's connect text color to match the figma design
- Changed font weight for the list of technologies

ichala reviewed (approved):
Hi @DianaBeki  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
![68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966](https://user-images.githubusercontent.com/85113567/184604589-430cf4bc-1d2e-4603-976a-7a48a899ea35.gif)

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the required changes :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

DianaBeki committed:
Add margin to bio class

DianaBeki committed:
code

DianaBeki committed:
Added images to image folder

DianaBeki committed:
Use main and section element to style in css

DianaBeki committed:
Style work section using flexbox layout

DianaBeki committed:
Add main tag in about section to style elements in css

DianaBeki committed:
Add background color property in about class to style

DianaBeki committed:
Update documentation

DianaBeki committed:
Add arrow class to style in css file

DianaBeki committed:
Add color  property and value to language class

Zuhaib042 commented on index.html:
- [ ] Kindly consider taking a `section` tag here as a container for the works section. it is due to following best practices, semantics and convention.

Zuhaib042 commented on index.html:
- [ ] Kindly consider taking a `section` tag here as a container for the about section. it is due to following best practices, semantics and convention.

Zuhaib042 commented on style.css:
- [ ] Kindly consider laying out elements in this container with `grid` instead of `flex` as it is a project requirement.

Zuhaib042 commented on index.html:
- [ ] There are some issues in your buttons, kindly consider applying following changes in all the buttons in your project to make it similar to the figma design which is a project requirement.
    -  the background of your buttons is not white as in the figma design. consider making it similar to the figma design.
    -  the hover state background color is not as suggested in the figma design. consider changing it also.
    -  In the figma design, there are button guidelines for the pressed state. you can achieve that by applying that background color with `:active` pseudoclass on your button.
 your button design
![qw](https://user-images.githubusercontent.com/96683843/219850288-273193a6-2f23-4571-bd58-c6a86d2600ac.jpg)

figma design guidelines
![zx](https://user-images.githubusercontent.com/96683843/219850290-58b3c27f-bdf6-4c66-b28f-e8fe22e42577.jpg)

Zuhaib042 commented on index.html:
- [ ] The font-size of this `p` tag in the figma is `16px` but in your design its 1.9rem which is double than required. kindly make adjustments here and try to make it similar to the figma design. 
your design
![fd](https://user-images.githubusercontent.com/96683843/219850507-f2f0592f-3e0e-4129-9ced-1972931eadd7.jpg)
figma
![sd](https://user-images.githubusercontent.com/96683843/219850532-7b958944-f6b9-4f77-9ba1-e3af5262e4d5.jpg)

Zuhaib042 commented on index.html:
- [ ] Kindly consider taking a `ul` for these icons instead of div. and then each icon would be inside an `li` and then `a` tag. this is to follow best practice and semantics.

Zuhaib042 commented on index.html:
- [ ] Kindly consider structuring your skills following way in order to follow best practices and semantics.
```
<ul>
  <li>Languages
    <ul>
      <li>HTML</li>
      <li>CSS</li>
      <li>Javascript</li>
    </ul>
  </li>
  <li>Frameworks</li>
  <li>Skills</li>
</ul>
```

Zuhaib042 reviewed (changes requested):
Hi @DianaBeki >, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
![close](https://media.giphy.com/media/z3piokwf0WPH81MOhu/giphy.gif)

## Highlights
✅ Correct Github flow
✅ Descriptive PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

DianaBeki committed:
Replace div with section

DianaBeki committed:
Replace button class with white background color

roshan-bajgain reviewed (approved):
Hi @DianaBeki, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
